sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/StorageApiWritesShardedRecords.java

```diff
@@ -652,6 +692,18 @@ public void process(
         }
         appendSplitDistribution.update(numAppends);
 
+        if (updatedSchemaReturned.get() != null) {
```

Comment:
Wondering if you can directly use the one on StreamWriter:
https://github.com/googleapis/java-bigquerystorage/blob/main/google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/StreamWriter.java#L157

Reply:
Maybe - is there an advantage to using that over the one in the response?

Reply:
There is a contract that it will only appear when a schema update happened (for once), which is our JsonWriter depending on, so you don't need to do the comparisons at all.

Reply:
Done